### ISSUE TITLE
Ability to choose build configurations for iOS.

### ISSUE THREAD

**Issue by f3mahmud** [microsoft/appcenter#1041]
A clear and concise description of what you want to happen:
It is good that we can choose the target/scheme already, but it would be very helpful if we choose the build configuration. Doesn't have to be a dropdown, but at least so that we can write it.

A clear and concise description of any alternative solutions or features you've considered:
Running the builds manually/away from appcenter

**Comment by jwargo** [comment]
@f3mahmud Thank you for this feedback, I will pass this feedback on to the service PM.

**Comment by f3mahmud** [comment]
Any updates on when this is going to be added? We have over 30 ios apps to migrate over to appcenter from hockeyapp and this feature is important for a lot of our projects, especially with the migration deadline coming up.

**Comment by nrajpurkar** [comment]
Hi @f3mahmud, I'm not really sure what you mean with this feature request? Can you provide more detail? 

In terms of this getting implemented - we monitor this repo and prioritize some feature requests each month. This gets posted in the iteration plan. We'll continue to track interest in this feature request and include that in our prioritization decisions. Can you tell me more about how lacking this feature is blocking your migration from HockeyApp to App Center?

**Comment by f3mahmud** [comment]
Hi @nrajpurkar , the step one of the link 
https://zeemee.engineering/how-to-set-up-multiple-schemes-configurations-in-xcode-for-your-react-native-ios-app-7da4b5237966 discusses setting up the configurations I am asking to be set up. The image attached shows a screenshot of xcode that should make it more comprehensible. Hopefully this clears the confusion.

Previously we used to run a separate CI/CD tool which would upload to hockeyapp. It was highly manipulatable so it worked. We make builds for qa which uses one set of endpoints/configurations and sometimes for production which uses different endpoints/configurations. We used to change the said build configuration in order to change the endpoints. With appcenter we cannot do that, atleast not without going into the code and changing that bit  and pushing. The attached image should further be able to clarify the part i m talking about. Without this feature we will have to stick to our other CI/CD tools and since hockeyapp will no longer be there, we wont have a choice but to look for alternatives. Appcenter has been good to us for the most part, but we are trying to stick to just one tool. So this is a bit of a blocker for us in order to choose appcenter for the long run. Hopefully this falls under one of your iterations in the near future.


<img width="1636" alt="Screen Shot 2019-10-18 at 3 01 26 PM" src="https://user-images.githubusercontent.com/50589311/67120936-5262e480-f1b8-11e9-834a-c24bd33a242b.png">
